### PR TITLE
[fx] Throw error when symbolically tracing control flow ops

### PR DESCRIPTION
**Pull request by angelayi**
Throws a better error when symbolically tracing control flow ops. Right now it throws an error when creating the function arguments.

### CODE REVIEW

**Review by zhxchen17 — DISMISSED** [review]

File: torch/fx/proxy.py [hunk 1]
```diff
@@ -334,6 +334,9 @@ def find_tracer(a):
         if torch.overrides.is_tensor_method_or_property(orig_method):
             return tracer.create_proxy('call_method', orig_method.__name__, args, kwargs)
         else:
+            if orig_method.__name__ in {"cond", "map"}:
```
Comment (zhxchen17) [inline torch/fx/proxy.py:337]: @angelayi are you sure this way of testing won't accidentally catch other function with the same name? could we add one unittest?

Comment (angelayi) [reply torch/fx/proxy.py]: Updated to now just check for pyoperators!

**Review by zhxchen17 — APPROVED** [review]

**Pull request closed without merging**